index.js

```diff
@@ -3,53 +3,271 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      type: "Vegetable",
+      price: 0.35,
     },
     {
       id: "002-carrot",
       name: "carrot",
+      type: "Vegetable",
       price: 0.35
     },
     {
       id: "003-apple",
       name: "apple",
+      type : "Fruits",
       price: 0.35
     },
     {
       id: "004-apricot",
       name: "apricot",
+      type : "Fruits",
       price: 0.35
     },
     {
       id: "005-avocado",
       name: "avocado",
+      type: "Vegetable",
       price: 0.35
     },
     {
       id: "006-bananas",
       name: "bananas",
+      type : "Fruits",
       price: 0.35
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
+      type: "Vegetable",
       price: 0.35
     },
     {
       id: "008-berry",
       name: "berry",
+      type : "Fruits",
       price: 0.35
     },
     {
       id: "009-blueberry",
       name: "blueberry",
+      type : "Fruits",
       price: 0.35
     },
     {
       id: "010-eggplant",
       name: "eggplant",
+      type: "Vegetable",
       price: 0.35
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+const headerStore = document.querySelector('#store')
+const storeItemList = document.querySelector('.store--item-list')
+const mainCart = document.querySelector('#cart')
+const cartItemListContainer = document.querySelector('.cart--item-list-container')
+const cartItemList = document.querySelector('.cart--item-list')
+const totalSection = document.querySelector('.total-section')
+const totalNbr = document.querySelector('.total-number')
+
+// for ul card and ul store
+const itemList = document.querySelector('.item-list')
+
+const objectNo = {
+   number : 0
+  }
+tempItems = state.items
```

Comment:
1. That's defaulting to use `var`

2. Reference variable. To make a copy, use `structuredClone(state.items)`